### DISCUSSION TITLE
Resources section in Documentation

### DISCUSSION THREAD

**Discussion by gregorwolf** [ui5-community/wdi5#335] [Ideas]
Hello wdi5 Team,

there are already some external resources regarding wdi5 available

- [wdi5 workshop at UI5con 2022](https://github.com/ui5-community/wdi5-workshop)
- [YouTube Videos](https://www.youtube.com/results?search_query=wdi5&sp=EgIIBQ%253D%253D)
- [SAP Community Blog Posts](https://blogs.sap.com/tag/wdi5/)
- [SAP Community Questions](https://answers.sap.com/topics/wdi5.html) (Only one right now)

It might be worth linking this resources from a page called "Resources & Support" like it's done in the CAP Documentation: [Resources & Support](https://cap.cloud.sap/docs/resources/)

CU
Gregor

**Comment by vobu** [comment]
that's a good idea and we'll definitely do it 👍 
There are quite some (smaller) docs task in the issues, so I'll round them up, do all together, including your suggestion!

**Comment by vobu** [comment]
done! → https://ui5-community.github.io/wdi5/#/resources
